casepro/msgs/views.py

```diff
@@ -186,7 +187,23 @@ def get_context_data(self, **kwargs):
             user = self.request.user
             page = int(self.request.GET.get('page', 1))
 
+            context['user_id'] = user.id
+
             search = self.derive_search()
+
+            # this is a refresh of messages
+            if self.request.GET.get('last_refresh', None):
+                new_messages = Message.search(org, user, search)
+
+                search['last_refresh'] = self.request.GET['last_refresh']
+                del search['after']
```

Comment:
@erikh360 thanks :) still unclear on why `after` and `last_refresh` are incompatible. Could you elaborate?